native/dgetrf.go

```diff
@@ -29,7 +29,7 @@ func (impl Implementation) Dgetrf(m, n int, a []float64, lda int, ipiv []int) (o
 		panic(badIpiv)
 	}
 	if m == 0 || n == 0 {
-		return
+		return false
```

Comment:
I'm just doing what it was doing before. If you think it should return true, I'm fine with that, but it's not what it was doing prior to this change.